src/app/modules/auth-private/sign-up/sign-up.graphql.ts

```diff
@@ -28,6 +28,23 @@ export const LocationListQuery = gql`
 	}
 `;
 
+export const ConsultationDeadlineQuery = gql`
```

Comment:
@Nishanthh We won't need this anymore, right?

Reply:
@avikt yes, not needed.